_posts/2021-12-31-a-year-in-review-2021.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: epic
+title: "A Year in Review: 2021"
+date: 2021-12-31
+categories: [team, engineering]
+author: artsy_engineering
+---
+
+Wow - what a year! Here are some notable achievements, failures, and learnings from Artsy Engineering in 2021...
+
+<!-- more -->
+
+## Success!
+
```

Comment:
(doesn't have to go right at the top, just thinking that it belongs in here somewhere!)

```suggestion

- Launched shipping integration with Arta
```